pyblish_qml/control.py

```diff
@@ -1197,7 +1198,7 @@ def iterator(plugins, context):
 
         message = test(**state)
         if message:
-            return log.error("Stopped due to %s" % message)
+            return stopiter_by_error("Stopped due to %s" % message)
```

Comment:
Nah, let's just return StopIteration. Do we still need the log message if StopIteration works?

Reply:
Okay ! Remove the bandage !